src/app/tests/suites/certification/Test_TC_DA_1_4.yaml

```diff
@@ -33,13 +33,9 @@ tests:
 
     - label: "Pre-Conditions"
       verification: |
```

Comment:
Some questions in general about these test cases:
- this is an entirely manual test case because it is for commissioners - where is this script used? Does this appear in the TH? Is this script just for QA use?
- The commissioners under test will almost certainly NOT show logs like in the examples. It's also permissible to just warn. So is having those logs there helpful if this is actually for cert? This seems like a lot of work to generate those logs when the test step could just say the exact same thing as the test plan

Reply:
The test shows up in the TH.
@Rajashreekalmane, can you address Cecille's other comments?